packages/builder/src/actions/roundApplication.ts

```diff
@@ -196,11 +196,7 @@ const applyToRound =
       })
       .on("transactionStatus", async (res) => {
         if (res.type === "success") {
-          dispatch({
-            type: ROUND_APPLICATION_LOADED,
-            roundAddress: roundId,
-            projectId: projectID,
-          });
+          console.log("Transaction Status Success", res.value);
```

Comment:
should we just check for error here since we do not do anything with "success" status?